config/maintainers.json

```diff
@@ -22,10 +22,10 @@
     },
     {
       "github_username": "NobbZ",
-      "show_on_website": false,
+      "show_on_website": true,
       "alumnus": false,
-      "name": null,
-      "bio": null,
+      "name": "Norbert Melzer",
+      "bio": "Found out about erlang via exercism, got hooked up and after fixing some trivial bugs, I have been added to the team,",
```

Comment:
So that's twice the length of @NobbZ in this place, but I think the one here is fine as a good first-pass. I was expecting neither linebreaks nor formatting to be respected.